website/docs/r/project.html.markdown

```diff
@@ -57,6 +57,7 @@ resource "mongodbatlas_project" "test" {
 * `name` - (Required) The name of the project you want to create.
 * `org_id` - (Required) The ID of the organization you want to create the project within.
 * `project_owner_id` - (Optional) Unique 24-hexadecimal digit string that identifies the Atlas user account to be granted the [Project Owner](https://docs.atlas.mongodb.com/reference/user-roles/#mongodb-authrole-Project-Owner) role on the specified project. If you set this parameter, it overrides the default value of the oldest [Organization Owner](https://docs.atlas.mongodb.com/reference/user-roles/#mongodb-authrole-Organization-Owner).
+* `tags` - (Optional) Map that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster. See [below](#tags).
```

Comment:
here

Reply:
Fixed in https://github.com/mongodb/terraform-provider-mongodbatlas/commit/51a0496a14a971c51daebda9bef68575e288dead